media/js/newsblur/models/folders.js

```diff
@@ -331,7 +331,7 @@ NEWSBLUR.Collections.Folders = Backbone.Collection.extend({
         url = url.replace('{folder_title}', Inflector.sluggify(this.options.title));
         console.log(['rss_url', this]);
         
-        return "https://" + NEWSBLUR.URLs.domain + url;
+        return document.location.protocol + "//" + NEWSBLUR.URLs.domain + url;
```

Comment:
It does not change the scheme.
It just uses the same protocol (https or http) by which website was reached.
e.g. even if you do not use HTTPS to access to the Newsblur you always get HTTPS url  in the 'Folder Settings - FOLDER RSS FEED ADDRESS'
![screenshot](https://cloud.githubusercontent.com/assets/233977/11651727/3212b354-9d93-11e5-82e7-289b6a728b24.png)